src/fundus/publishers/kr/mbn.py

```diff
@@ -0,0 +1,58 @@
+import re
+from datetime import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, Image, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+    image_extraction,
+)
+
+
+class MBNParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = XPath("//div[@class='mid_title']//div")
+        _paragraph_selector = XPath(
+            "//div[@itemprop='articleBody']//p | //div[@itemprop='articleBody']//div[normalize-space(text())]"
+        )
+
+        @attribute
+        def body(self) -> Optional[ArticleBody]:
+            return extract_article_body_with_selector(
+                self.precomputed.doc,
+                paragraph_selector=self._paragraph_selector,
+                summary_selector=self._summary_selector,
+            )
+
+        @attribute
+        def authors(self) -> List[str]:
+            return generic_author_parsing(self.precomputed.ld.get_value_by_key_path(["NewsArticle", "author"]))
```

Comment:
```suggestion
            return generic_author_parsing(self.precomputed.ld.xpath_search("//NewsArticle//author"))
```
Please replace all instances of `get_value_by_key_path` with `xpath_search` as the first is deprecated and going to be removed in one of the newer versions (then there will also not be any issues with the testing). The switch is pretty straightforward, you can just prepend all values of the list with a double forward slash and combine them into one string as the XPath.